articles/install-guide/pyinstall.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Develop with Q# + Python
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.python
+---
+
+# Develop with Q# + Python
+
+This is the installation guide for the QDK environment to program hybrid Q# and Python programs.
```

Comment:
Suggest changing this to more active language:

"Install the QDK for developing Q# operations called from a Python host program."

Reply:
Or "Install the QDK to develop Python host programs to call Q# operations"

Reply:
Done in 467bf40